### ISSUE TITLE
Some Metadata Menu plugin's icons visibility are not well.

### ISSUE THREAD

**Issue by ertugruluyar-github** [nickmilo/LYT-Mode#56]
![image](https://user-images.githubusercontent.com/70096810/226115415-3a846692-1632-4190-b4b1-f64fb23de639.png)

**Comment by nickmilo** [comment]
Could you include a screenshot of the default theme as well?

**Comment by ertugruluyar-github** [comment]
Of course, first one default theme and the second one minimal theme. By the way, I learned lots of things about zettelkasten and note taking from you, thank you so much.
Default Theme:
![image](https://user-images.githubusercontent.com/70096810/227517184-08f1cdec-189b-40e7-8525-a813001e639c.png)
Minimal Theme:
![image](https://user-images.githubusercontent.com/70096810/227517797-37e0a07c-3d4f-4c9e-afbe-ed697f1eb882.png)
